ur_robot_driver/include/ur_robot_driver/ur/ur_driver.h

```diff
@@ -191,10 +197,11 @@ class UrDriver
 
   std::thread watchdog_thread_;
   bool reverse_interface_active_;
-  uint32_t reverse_port_;
   std::function<void(bool)> handle_program_state_;
 
   std::string robot_ip_;
+  int reverse_port_;
+  int script_sender_port_;
```

Comment:
I would prefer keeping this as `uint32_t`. The conversion then has to be done when reading in ROS parameters.